test/services/entity_builder.js

```diff
@@ -157,9 +157,20 @@ describe('Entity Builder', () => {
           .and.have.nested.property('errors[0].dataPath', '.geoJson.coordinates');
       });
 
-      it('throws if accessLevel not positive integer', () => {
+      it('throws if timestamp is not a positive integer', () => {
+        let brokenEvent = put('content.idData.timestamp', exampleEvent, '1');
+        expect(() => entityBuilder.validateEvent(brokenEvent)).to.throw(ValidationError);
+        brokenEvent = put('content.idData.timestamp', exampleEvent, 1.1);
+        expect(() => entityBuilder.validateEvent(brokenEvent)).to.throw(ValidationError);
+        brokenEvent = put('content.idData.timestamp', exampleEvent, -1);
+        expect(() => entityBuilder.validateEvent(brokenEvent)).to.throw(ValidationError);
+      });
+
+      it('throws if accessLevel is not a positive integer', () => {
         let brokenEvent = put('content.idData.accessLevel', exampleEvent, 1.1);
         expect(() => entityBuilder.validateEvent(brokenEvent)).to.throw(ValidationError);
+        brokenEvent = put('content.idData.accessLevel', exampleEvent, 1.1);
+        expect(() => entityBuilder.validateEvent(brokenEvent)).to.throw(ValidationError);
```

Comment:
I'd add a positive case to see if it does not always throw. Just a sanity check.